index.html

```diff
@@ -162,34 +171,39 @@ <h3> Languages</h3>
       </details>
 
       <details class="abitity">
-        <summary>
+        <summary >
           <h3> Frameworks</h3>
-          <img src="images/arrow.svg" alt="arrow">
+          <img class="arrow" src="images/arrow.svg" alt="arrow">
         </summary>
       </details>
 
       <details class="abitity">
-        <summary>
+        <summary >
           <h3>Skills</h3>

```

Comment:
- [ ] Consider applying the correct color for this `h3` element so that it matches your design template. From your design, I could see that this `#344563;` is the color for this `h3` element on the desktop version. Please check your template and make corrections.